### ISSUE TITLE
Collect enough privacy flag data in a centralized location

### ISSUE THREAD

**Issue by stanleymarkman** [privacy-tech-lab/optmeowt-browser-extension#156]
Not sure yet where we'll collect this data, but there should be a representative sample of privacy flags collected.

My initial idea is just to install the extension on my regular web browser to collect enough real data via regular browsing.

**Comment by SebastianZimmeck** [comment]
> My initial idea is just to install the extension on my regular web browser to collect enough real data via regular browsing.

Yes, that is a good idea! Initially, just browsing and getting a handle is sufficient. Later on, we can come up with a more systematic approach.

I am transferring this issue to the gpc-paper repo (to which I invited you and @kbeliauski). Feel free to store data from your browsing there.
